test/e2e/apps/deployment.go

```diff
@@ -30,6 +30,7 @@ import (
 
 	"encoding/json"
```

Comment:
While touching, can you move this import up to built-ins?

Reply:
Moved